models/public/yolo-v2-tiny-tf/pre-convert.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import subprocess
+import sys
+
+from pathlib import Path
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('download_dir', type=Path)
+    parser.add_argument('convert_dir', type=Path)
```

Comment:
These arguments are not `download_dir` and `convert_dir` (well, the last one may be) from very this script perspective

Reply:
What do you mean? `download_dir` is the directory with the downloaded files and `convert_dir` is the directory to put conversion results in.

Reply:
Script knows nothing about downloading. It is awaiting "model in - model out" parameters

Reply:
I'm not sure I agree, but it doesn't really matter. Renamed to `input_dir`/`output_dir`, respectively.